zipkin-storage/cassandra/src/main/java/zipkin2/storage/cassandra/CassandraSpanStore.java

```diff
@@ -106,7 +106,7 @@ static SelectTraceIdsFromSpan.Factory initialiseSelectTraceIdsFromSpan(CqlSessio
     try {
       return new SelectTraceIdsFromSpan.Factory(session);
     } catch (DriverException ex) {
-      LOG.warn("failed to prepare annotation_query index statements: " + ex.getMessage());
+      LOG.warn("failed to prepare annotation_query index statements: {}", ex.getMessage(), ex);
```

Comment:
Would you indeed want the full stacktrace logged here? Our best practices recipes does that by default, but welcome to revert.
```suggestion
      LOG.warn("failed to prepare annotation_query index statements: {}", ex.getMessage());
```

Reply:
works fine, thanks